src/serverless.js

```diff
@@ -7,17 +7,41 @@ const fs = require("fs");
 const { createOrUpdateMetaRole } = require("./utils");
 
 class LambdaCron extends Component {
+  validate(inputs) {
+    if (!inputs.schedule) {
+      throw new Error(
+        "Input 'schedule' is required. Please see README: https://git.io/JJWW0"
+      );
+    }
+    const { schedule } = inputs.schedule;
+
+    let valid = false;
+
+    // Check for a cron expression
+    const cronRegex = /^((((\d+,)+\d+|(\d+(\/|-|#)\d+)|\d+L?|\*(\/\d+)?|L(-\d+)?|\?|[A-Z]{3}(-[A-Z]{3})?) ?){5,7})$/;
```

Comment:
Close! Just need to add `cron` here in the front, so it will match valid input such as:
`cron(15 10 * * ? *)`
I think this is valid:
```suggestion
    const cronRegex = /^cron\(((((\d+,)+\d+|(\d+(\/|-|#)\d+)|\d+L?|\*(\/\d+)?|L(-\d+)?|\?|[A-Z]{3}(-[A-Z]{3})?) ?){5,7})\)$/;
```
It satisfies the test cases that AWS has on their docs page:
![image](https://user-images.githubusercontent.com/1598537/94965909-f32ddb00-04c1-11eb-88e0-2223f516e6a2.png)

Reply:
Ah yes! I didn't see the cron part. Will do it rn!

Reply:
No rush, the PR looks great otherwise!

Reply:
Okay! It's fixed already.